speckle/resources/objects.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, session, basepath):
                 'method': 'PUT'
             }
         })
+           
+        self.schema = SpeckleObject
```

Comment:
Good catch! Should've definitely not PR'd this at midnight :laughing: Got a bit too excited.